src/style.css

```diff
@@ -1 +1,62 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  padding: 0;
+  margin: 20px 0;
+
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px #3d4e611a;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.form__input--primary {
+  height: 70px;
+  background-size: 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.form__input--secondary {
+  height: 42px;
+  background-size: 11px;
+  background-position: 13px center;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.form__input::placeholder {
+  color: #3d4e61;
+}
+
+.form__input:hover {
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.25);
+}
+
+.form__input:focus {
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.25);
+  text-shadow: 0 1px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 900;
```

Comment:
You're changing the font weight on focus. This might lead to a visual shift in your layout when the input is focused. It's not a common practice to change font weight on focus or hover states. Consider removing it.